src/utils/ConfigManager.ts

```diff
@@ -38,7 +38,11 @@ export default class ConfigManager {
 
 		if (this.#configFile) {
 			// Relative paths are needed to make it work on Windows
-			const configFilePath = this.#resolveModulePaths(this.#configFile);
+
+			const configFilePath = path.isAbsolute(this.#configFile) ?
+				this.#configFile : // If it's an absolute path, transform to POSIX format
```

Comment:
This one allows inclusion by absolute path that might not be within the project itself

Reply:
Not planned

Reply:
Config files may be located anywhere on the system if the path is provided via CLI. Just like for the ui5.yaml in case of UI5 Tooling